### DISCUSSION TITLE
2.4 API Documentation

### DISCUSSION THREAD

**Discussion by marcinwiktor** [cachethq/cachet#4375] [Q&A]
Hi, 
I know that API will be modified in 3.0 version but I'd still like to start using cachet now. Unfortunately I've searched through github and docs but I can't find any working link to API documentation. There is only an articile on how to authorize. Is there any (even old) version of this documentation still available?

**Comment by jbrooksuk** [comment]
Sorry about that! I hadn't gotten around to copying all documentation yet. I've made a basic start of this https://docs.cachethq.io/api/
